README.md

```diff
@@ -15,36 +16,6 @@ Add the following to your sbt `project/plugins.sbt` file:
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
 ```
 
-**Note**: Both the plugin organization and the name has changed in 0.4.0.
-
-You will need to add the following to your `project/build.properties` file if you have multiple versions of sbt installed
-
-    sbt.version=0.13.16
-
-Be sure to use the [latest launcher](http://www.scala-sbt.org/download.html)
-
-## Resolving bintray artifacts
```

Comment:
This might be a useful info for the consumer of Bintray-published libraries.

Reply:
okay, I pushed a second commit that restores that info, in somewhat briefer form